tests/pytests/test_followhashes.py

```diff
@@ -591,7 +591,7 @@ def testWrongFieldType(env):
 
     res_actual = env.cmd('FT.INFO idx')
     res_actual = {res_actual[i]: res_actual[i + 1] for i in range(0, len(res_actual), 2)}
-    env.assertEqual(str(res_actual['hash_indexing_failures']), '1')
+    env.assertEqual(str(res_actual['indexing_failures']), '1')
```

Comment:
Did we change the name? its a breaking change...

Reply:
This can be reverted but the terminology is not accurate anymore.
@emmanuelkeller , WDYT?